addons/functions/scopebox/fn_scopebox.sqf

```diff
@@ -1,69 +1,70 @@
 /* ----------------------------------------------------------------------------
```

Comment:
Isn't this not the subject of this enhancement?

Reply:
Secret fix to some formatting problems in the code, I did test it still works afterwards though.